packages/extension-ui/src/Popup/Forget.tsx

```diff
@@ -45,12 +46,12 @@ function Forget({
   const _goTo = useCallback((path: string) => () => onAction(path), [onAction]);
 
   const _onClick = useCallback((): void => {
+    onAction('/');
```

Comment:
Why this order? Shouldn't we first show the notification and then redirect? In some cases like very fast machine might it redirect so fast it won't "catch" the notification?